src/akkudoktoreos/optimization/genetic.py

```diff
@@ -488,10 +488,10 @@ def optimierung_ems(
         )
 
         # Initialize the inverter and energy management system
-        wr = Wechselrichter(parameters.wechselrichter, akku)
+        wr = Inverter(parameters.inverter, akku)
```

Comment:
variable name could be translated

Reply:
yes, but this is the pr for the inverter file. I cant translate all parts of all files. that would be an endless rabbit hole. 
imo battery must come next.

Reply:
done anyway ;-)